src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -110,6 +110,32 @@
   col-sm input-container
 {%- endblock form_group_class %}
 
-{% block form_row_class -%}
-  form-group row{% if row_attr.class is defined %} {{ row_attr.class }}{% endif %}
-{%- endblock form_row_class %}
+{% block form_row_attr -%}
+  {% set classes = 'form-group row' %}
+  {% if (not compound or force_error|default(false)) and not valid %}
+    {% set classes = classes ~ ' has-error' %}
+  {% endif %}
+
+  {% if attr.visible is defined and not attr.visible %}
+    {% set classes = classes ~ ' d-none' %}
+  {% endif %}
+
+  {% if row_attr.class is defined %}
+    {% set classes = classes ~ ' ' ~ row_attr.class  %}
+  {% endif %}
+
+  {% set row_attr = row_attr|merge({'class': classes}) %}
+
+  {% for key, rowAttr in row_attr %} {{ key }}="{{ rowAttr }}" {% endfor %}
+{%- endblock form_row_attr %}
```

Comment:
However, this whole block should be put in the `prestashop_ui_kit_base.html.twig` form theme so that this new attribute block can also be used in the other layout (which is used for product page)

The only diff will be that the base layout doesn't use the same class for its row it uses `form-group` instead of `form-group row` This is where keeping a separated block `form_row_class` will come in handy, the base layout can define it And this layout can simply override it to include its different classes The `form_row_attr` will remain common and the same for both themes